### ISSUE TITLE
announce dates and theme

### ISSUE THREAD

**Issue by dcwalk** [ournetworks/2019#10]
This relies on #8 and #9

Things to mark this as done:
- [x] update website (https://github.com/ournetworks/ournetworks.ca/pull/91)
- [x] announce on social media
- [x] add dates to website (https://github.com/ournetworks/ournetworks.ca/pull/94)
- [x] clean up social media
  - [youtube](https://www.youtube.com/channel/UCudGRFTjVGsrKF0h27GER2g) background, logo
  - [twitter](https://twitter.com/_ournetworks) background, description, pinned tweet, logo
- [x] re-announce CFP on mailing list
- [x] announce on social media
  - [x] [twitter](https://twitter.com/_ournetworks), description, pinned tweet
  - [x] SSB
- [x] add CFP and open hours to website (see #25)
- [x] share new details before 2nd open hour

**Comment by dcwalk** [comment]
Let's wait on announce until end of this week -- March 8ish? We should hopefully hear from first venue then :)

**Comment by dcwalk** [comment]
This has been done as done as can be. Closing and will spawn new issue for announcing registration launch
